config/samples/kafkacluster-with-istio.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   headlessServiceEnabled: false
   ingressController: "istioingress"
   istioControlPlane:
-    name: icp-v111x-sample
+    name: icp-v113x-sample
```

Comment:
I think we should add a comment here explaining that this should be the name of the `IstioControlPlane` custom resource and example of which can be found here https://github.com/banzaicloud/istio-operator/blob/v2.13.0-rc.0/config/samples/servicemesh_v1alpha1_istiocontrolplane.yaml